piper/base/executors/clip.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import FastAPIExecutor
```

Comment:
It's better to use full paths

from piper.base.executors import FastAPIExecutor